boards/xtensa/nxp_adsp_rt685/nxp_adsp_rt685.dts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2023 Google LLC.
```

Comment:
Why do you use the Google copyright?

It should be NXP, or if you used the rt5xx as example, you should add NXP copyright also.

Reply:
That was done in a following commit.

Reply:
yes, don't add a new commit to fix something from a commit that was not merged.

Reply:
Commit history redone - no fixup commits.